CONTRIBUTING.md

```diff
@@ -16,72 +17,93 @@ First of all, thanks for taking the time to contribute! This project can only gr
+  <li><b>original</b>: the original logo. Can contain multiple colors. <a href="https://github.com/devicons/devicon/blob/master/icons/amazonwebservices/amazonwebservices-original.svg"> Example </a> </li>
   <li><b>original-wordmark</b>: similar to the above but must contain the name of the technology.<a href="https://github.com/devicons/devicon/blob/master/icons/amazonwebservices/amazonwebservices-original-wordmark.svg"> Example </a></li>
   <li><b>plain</b>: a one-color version of the original logo.<a href="https://github.com/devicons/devicon/blob/master/icons/android/android-plain.svg"> Example </a></li>
   <li><b>plain-wordmark</b>: a one-color version of the original logo but with wordmark.<a href="https://github.com/devicons/devicon/blob/master/icons/android/android-plain-wordmark.svg"> Example </a></li>
   <li><b>line</b>: a one-color, line version of the original logo.<a href="https://github.com/devicons/devicon/blob/master/icons/apache/apache-line.svg"> Example </a></li>
   <li><b>line-wordmark</b>: a one-color, line version of the original logo but with wordmark.<a href="https://github.com/devicons/devicon/blob/master/icons/apache/apache-line-wordmark.svg"> Example </a></li>
 </ul>
-<p>
-It is not mandatory to have 6 versions for each icon. An icon can only have one or two versions available. Just keep in mind that the minimum is 1 and the maximum 6 (for now). You must also have at least one version that can be make into an icon.
-</p>
-<p>
-The <b>plain</b> and <b>line</b> versions (with or without wordmark) are designed to be available in the final icon font.
-</p>
-<p>
-The <b>original</b> version are only available in svg format, so they do not need to be as simple and can contain numerous colors.
-</p>
-<p>
-Some icons are really simple (like the Apple one), so the original version can be used as the plain version and as the icon font. In this case, you'll only need to make only one of the version (either "original" or "plain"). You can then add an alias in the <code>devicon.json</code> so they can be found with either the "original" or "plain" naming convention. Note that this only applies to font icon versions only, not the SVG versions. SVG versions don't need aliases.
-</p>
+<p> Notes <p>
+<<<<<<< HEAD
+<ol>  
+  <li>
+    It is not mandatory to have 6 versions for each icon. An icon can only have one or two versions available. Just keep in mind that the minimum is 1 and the maximum 6 (for now). You must also have at least one version that can be make into an icon.
+=======
+<ul>  
+  <li>
+    You don't need to have 6 versions for each icon. An icon can only have one or two versions available. Just keep in mind that the minimum is 1 and the maximum 6 (for now). You must also have at least one version that can be make into an icon.
+>>>>>>> 09c3b64... Updated CONTRIBUTING
+  </li>
```

Comment:
There is something wrong, looks like a merge error?

Reply:
Huh, that's interesting. I did have to cherry pick a commit from another branch (the update imgur ref) so that must have been the cause. I don't understand how my editor would let that pass by me. I'll fix it